metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -144,6 +144,24 @@ object MetalsEnrichments extends DecorateAsJava with DecorateAsScala {
     }
   }
 
+  implicit class XtensionDocumentSymbol(symbol: Seq[l.DocumentSymbol]) {
+
+    def toSymbolInformation(uri: String): List[l.SymbolInformation] = {
+      val buf = List.newBuilder[l.SymbolInformation]
+      def loop(s: l.DocumentSymbol): Unit = {
+        buf += new l.SymbolInformation(
+          s.getName,
+          s.getKind,
+          new l.Location(uri, s.getRange),
+          s.getDetail
```

Comment:
I don't love the implicit detail <~> containerName correspondence, but since the LSP spec doesn't say much about it, I guess it's fine.

Let's keep in mind not to add fancier stuff to details, or this conversion will break.

Reply:
`containerName` now contains the enclosing fully qualified symbol which makes the output more readable in at least vim

<img width="724" alt="vim-document-symbol" src="https://user-images.githubusercontent.com/1408093/50220925-ed6e1080-0393-11e9-8db9-66015f3646e0.png">